plutus-playground-client/src/Types.purs

```diff
@@ -200,6 +201,22 @@ data ActionEvent
   | RemoveAction Int
   | SetWaitTime Int Int
 
+data DragAndDropEventType
+  = DragStart
+  | DragEnd
+  | DragEnter
+  | DragOver
+  | DragLeave
+  | Drop
+
+instance showDragAndDropEventType :: Show DragAndDropEventType where
+  show DragStart = "DragStart"
+  show DragEnd = "DragEnd"
+  show DragEnter = "DragEnter"
+  show DragOver = "DragOver"
+  show DragLeave = "DragLeave"
+  show Drop = "Drop"
```

Comment:
can't be derived?

Reply:
If derived it would be `DragAndDropEventType.Drop` and so on. I want just the last bit. I suppose I could drop up to the last `.`, or drop the static prefix. 🤔